client/src/pages/ProfilePage.jsx

```diff
@@ -0,0 +1,447 @@
+                  errors.confirmation
+                }
+              >
+                Submit
+              </button>
+            </div>
+          </Form>
+        )}
+      </Formik>
+
+      <Formik
+        initialValues={{
+          newEmail: '',
+          password: '',
+        }}
+        validateOnMount={true}
+        onSubmit={({ newEmail, password }, formikHelpers) => {
+          formikHelpers.setSubmitting(true);
+
+          userService
+            .changeEmail({ userId, newEmail, password })
+            .then((response) => {
+              alert(response);
+              checkAuth();
+              formikHelpers.resetForm();
+              navigate('/login');
+            })
+            .catch((error) => {
+              if (error.message) {
+                setError(error.message);
+              }
+
+              if (!error.response?.data) {
+                return;
+              }
+
+              const { errors, message } = error.response.data;
+
+              formikHelpers.setFieldError('newEmail', errors?.newEmail);
+              formikHelpers.setFieldError('password', errors?.password);
+
+              if (message) {
+                setError(message);
+              }
+            })
+            .finally(() => {
+              formikHelpers.setSubmitting(false);
+            });
+        }}
+      >
+        {({ touched, errors, isSubmitting }) => (
+          <Form className="box">
+            <h1 className="title">Change Email</h1>
+            <div className="field">
+              <label htmlFor="newEmail" className="label">
+                New email
+              </label>
+
+              <div className="control has-icons-left has-icons-right">
+                <Field
+                  validate={validateEmail}
+                  name="newEmail"
+                  type="email"
+                  id="newEmail"
+                  placeholder="e.g. bobsmith@gmail.com"
+                  className={cn('input', {
+                    'is-danger': touched.newEmail && errors.newEmail,
+                  })}
+                />
+
+                <span className="icon is-small is-left">
+                  <i className="fa fa-envelope"></i>
+                </span>
+
+                {touched.newEmail && errors.newEmail && (
+                  <span className="icon is-small is-right has-text-danger">
+                    <i className="fas fa-exclamation-triangle"></i>
+                  </span>
+                )}
+              </div>
+
+              {touched.newEmail && errors.newEmail && (
+                <p className="help is-danger">{errors.newEmail}</p>
+              )}
+            </div>
+            <div className="field">
+              <label htmlFor="password" className="label">
+                Password
+              </label>
+
+              <div className="control has-icons-left has-icons-right">
+                <Field
+                  // validate={validatePassword}
```

Comment:
The `validatePassword` function is commented out for the password field in the email change form. If password validation is required, consider uncommenting this line.